scripts/makeLegacyPlots_postFit.py

```diff
@@ -0,0 +1,935 @@
+        pad1 = TPad ("pad1", "pad1", 0, 0.31, 1, 1.0)
+        pad1.SetFrameLineWidth(3)
+        pad1.SetLeftMargin(0.12)
+        pad1.SetBottomMargin(0.02)
+        pad1.SetTopMargin(0.055)
+        pad1.Draw()
+    else:
+        pad1 = TPad ("pad1", "pad1", 0, 0.0, 1.0, 1.0)
+        pad1.SetFrameLineWidth(3)
+        pad1.SetLeftMargin(0.12)
+        pad1.SetBottomMargin(0.12)
+        pad1.SetTopMargin(0.055)
+        pad1.Draw()
+
+    pad1.cd()
+    if args.log: pad1.SetLogy()
+
+
+    ######################### USER CONFIGURATION ####################
+    if args.postFitFile == '': 
+        print('** ERROR: no file containing postfit shapes and nuisances specified')
+        print('** EXITING')
+        exit()
+
+    # the channel-bin assignement is done with ROOT's alphabetical order (alphabetical order with precedence to capital letters)
+    if args.channel == 'ETau':
+        if args.year == '2016':
+            if "1b1j" in args.sel:        combine_ch = 'ch55'
+            elif "2b0j" in args.sel:      combine_ch = 'ch64'
+            elif "boosted" in args.sel:   combine_ch = 'ch1'
+            elif "GGFclass" in args.sel:  combine_ch = 'ch19'
+            elif "VBFclass" in args.sel:  combine_ch = 'ch37'
+            elif "DYclass" in args.sel:   combine_ch = 'ch10'
+            elif "ttHclass" in args.sel:  combine_ch = 'ch46'
+            elif "TTclass" in args.sel:   combine_ch = 'ch28'
+            else: print('** ERROR: incorrect region name'); print('** EXITING'); exit()
+        elif args.year == '2017':
+            if "1b1j" in args.sel:        combine_ch = 'ch56'
+            elif "2b0j" in args.sel:      combine_ch = 'ch65'
+            elif "boosted" in args.sel:   combine_ch = 'ch2'
+            elif "GGFclass" in args.sel:  combine_ch = 'ch20'
+            elif "VBFclass" in args.sel:  combine_ch = 'ch38'
+            elif "DYclass" in args.sel:   combine_ch = 'ch11'
+            elif "ttHclass" in args.sel:  combine_ch = 'ch47'
+            elif "TTclass" in args.sel:   combine_ch = 'ch29'
+            else: print('** ERROR: incorrect region name'); print('** EXITING'); exit()
+        elif args.year == '2018':
+            if "1b1j" in args.sel:        combine_ch = 'ch57'
+            elif "2b0j" in args.sel:      combine_ch = 'ch66'
+            elif "boosted" in args.sel:   combine_ch = 'ch3'
+            elif "GGFclass" in args.sel:  combine_ch = 'ch21'
+            elif "VBFclass" in args.sel:  combine_ch = 'ch39'
+            elif "DYclass" in args.sel:   combine_ch = 'ch12'
+            elif "ttHclass" in args.sel:  combine_ch = 'ch48'
+            elif "TTclass" in args.sel:   combine_ch = 'ch30'
+            else: print('** ERROR: incorrect region name'); print('** EXITING'); exit()
+        else: print('** ERROR: incorrect year'); print('** EXITING'); exit()
+    elif args.channel == 'MuTau':
+        if args.year == '2016':
+            if "1b1j" in args.sel:        combine_ch = 'ch1'#'ch58'
```

Comment:
is this `ch1` or `ch58` ? there is another `ch1` for boosted etau 2016

Reply:
this is like this for the testing, `ch58` is the correct one and will be uncommented as soon as the `fitDiagnostic.root` for the full analysis is available

Reply:
But if you already know the correct name why do you need the root file? what do you want to test?

Reply:
because the tests are done with a dummy file that contains only 3 channels that are the 1b1j - MuTau - 2016/7/8 channels. So the code already contains the full naming as it will be for the full analysis, but to test that the code is working I need the dummy naming for the dummy file

Reply:
ok I understand! but since this is going to be pushed&merged please put the correct number here.